### DISCUSSION TITLE
A bit confuse on how to use sakai themes component to build my project

### DISCUSSION THREAD

**Discussion by VivekParmar-18** [primefaces/community#1494] [PrimeNG Templates]
i have used login component and trying to toast the message when there is wrong credentials entered. while doing that i got error of unknown tag
even if i have imported MessageDemoModule
X [ERROR] NG8001: 'p-toast' is not a known element:
1. If 'p-toast' is an Angular component, then verify that it is part of this module.
2. If 'p-toast' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. [plugin angular-compiler]

    src/app/components/login/login.component.html:11:16:
      11 │                 <p-toast key="tst"></p-toast>
         ╵                 ~~~~~~~~~~~~~~~~~~~

  Error occurs in the template of component LoginComponent.

    src/app/components/login/login.component.ts:9:17:
      9 │     templateUrl: './login.component.html',
        ╵                  ~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by cetincakiroglu** [comment]
Hi,

It seems that there is a missing import in the related module of this error. Could you please try to import `ToastModule`?
